third_party/amd/lib/TritonAMDGPUToLLVM/DotOpToLLVM/MFMA.cpp

```diff
@@ -632,18 +637,22 @@ struct ScaledDotOpMFMAConversionHelper : DotOpMFMAConversionHelper {
           }
           acc = zeroAuxiliarBlocks(subBlocks, acc);
           for (int k = 0; k < numRepK; k++) {
-            for (int kPack = 0; kPack < kWidth / kBase; ++kPack) {
+            for (int kPack = 0; kPack < aKWidth / aKBase; ++kPack) {
               acc = mfmaLayout.getIsTransposed()
-                        ? generateScaledMFMAOp(maybeMfmaIntrinsic.value(),
+                        ? generateScaledMFMAOp(intrinsicName,
```

Comment:
Nit, this ternary expression is spanning multiple lines now. Better to change it into a normal if statement.